administration/src/bp-modules/applications/ApplicationCard.tsx

```diff
@@ -130,79 +143,93 @@ const ApplicationCard = ({
+        collapseProps={{ isOpen: isExpanded, onToggle: () => setIsExpanded(!isExpanded), keepChildrenMounted: true }}
+        collapsible
+        $hideInPrintMode={!isSelectedForPrint}>
+        <SectionCard>
+          <SectionCardHeader>
+            <NoteButton onClick={() => setOpenNoteDialog(true)} intent='none' icon='annotation'>
+              Notiz anzeigen
+            </NoteButton>
+            {withdrawalDate && (
+              <WithdrawAlert intent='warning'>
+                Der Antrag wurde vom Antragsteller am {formatDateWithTimezone(withdrawalDate, config.timezone)}{' '}
+                zurückgezogen. <br />
+                Bitte löschen Sie den Antrag zeitnah.
+              </WithdrawAlert>
+            )}
+          </SectionCardHeader>
+          <JsonFieldView
+            jsonField={jsonField}
+            baseUrl={baseUrl}
+            key={0}
+            hierarchyIndex={0}
+            attachmentAccessible
+            expandedRoot={false}
+          />
+        </SectionCard>
+        <SectionCard>
+          <VerificationsView verifications={application.verifications} />
+        </SectionCard>
+        <SectionCard>
+          <ButtonContainer>
+            <Tooltip
+              disabled={!!createCardQuery}
+              content={
+                'Es existiert kein passendes Mapping, um aus diesem Antrag das Kartenformular vollständig auszufüllen.'
+              }>
+              <PrintAwareAnchorButton
+                disabled={!createCardQuery}
+                href={createCardQuery ? `./cards/add${createCardQuery}` : undefined}
+                icon='id-number'
+                intent='primary'>
+                Karte erstellen
+              </PrintAwareAnchorButton>
+            </Tooltip>
+            <PrintAwareButton onClick={() => setDeleteDialogOpen(true)} intent='danger' icon='trash'>
+              Antrag löschen
+            </PrintAwareButton>
+            <PrintAwareButton onClick={() => printApplicationById(id)} intent='none' icon='print'>
+              PDF exportieren
+            </PrintAwareButton>
+            <CollapseIcon
+              icon={'chevron-up'}
+              onClick={() => setIsExpanded(!isExpanded)}
+              style={{ marginLeft: 'auto' }}
+            />
+          </ButtonContainer>
+          <Alert
+            cancelButtonText='Abbrechen'
+            confirmButtonText='Antrag löschen'
+            icon='trash'
+            intent='danger'
+            isOpen={deleteDialogOpen}
+            loading={loading}
+            onCancel={() => setDeleteDialogOpen(false)}
+            onConfirm={() => deleteApplication({ variables: { applicationId: application.id } })}>
+            <p>Möchten Sie den Antrag unwiderruflich löschen?</p>
+          </Alert>
+        </SectionCard>
+      </ApplicationViewCard>
+      {openNoteDialog && (
+        <NoteDialogController application={application} isOpen={openNoteDialog} onOpenNoteDialog={setOpenNoteDialog} />
+      )}
```

Comment:
Is it possible to move this inside the ApplicationViewCard? Then we don't need the fragment (which brings a pretty bad diff in this PR + more indents)